aten/src/ATen/function_wrapper.py

```diff
@@ -887,7 +889,10 @@ def get_return_types(option):
 
     def format_return_type(return_types):
         # type: (List[ReturnType]) -> str
-        if len(return_types) == 1:
+        len_return_types = len(return_types)
+        if len_return_types == 0:
+            return "void"
+        if len_return_types == 1:
```

Comment:
nit (this means no change is needed here, this is just some tips for the next time you write some code): len should be O(1) on lists in Python, so you don't have to assign it to a variable. Also, intent is clearer if you use `elif` for the second if statement.